listenbrainz/listen.py

```diff
@@ -99,10 +99,12 @@ def __init__(self, user_id=None, user_name=None, timestamp=None, artist_msid=Non
     @classmethod
     def from_json(cls, j):
         """Factory to make Listen() objects from a dict"""
+        if j['listened_at']:
```

Comment:
I would like a test for `from_json` where we create pass a json file with `playing_now` set to true and check that the timestamp is None and then pass a json file for a normal listen (non `playing_now`) and check that the timestamp is valid in the Listen object. 

You can add this test to `test_listen.py`.